addons/web/static/tests/views/list_view_tests.js

```diff
@@ -3497,6 +3497,29 @@ QUnit.module("Views", (hooks) => {
         );
     });
 
+    QUnit.test("groups can be sorted when list is grouped by date with granularity", async function (assert) {
```

Comment:
line too long, please consider using the web tooling (`./addons/web/tooling/enable.sh`). It enables a pre-commit hook that lints and prettifies the code